app/pages/index.tsx

```diff
@@ -103,6 +103,20 @@ const StandardRealmsSplashContainer = styled.div`
   }
 `;
 
+const WhatsNew = styled.div`
+  background: #38598a;
+  color: #ffffff;
+  font-size: 18px;
+  margin-bottom: 55px;
+  margin-top: 55px;
+  padding-bottom: 55px;
+  padding-top: 55px;
+  weight: 400px h2 {
```

Comment:
Is the weight here getting applied without the semicolon? I thought that broke the css but not sure

Reply:
fixed, good catch